### DISCUSSION TITLE
Deploying doesn't work. (https://vercel.com/templates/next.js/blog-next-sanity)

### DISCUSSION THREAD

**Discussion by gllrmzndm** [vercel/community#1449] [Help] [answered]
### Summary

Deploying doesn't work.

![Web capture_5-2-2023_0525_vercel com](https://user-images.githubusercontent.com/43880437/216794988-2c9dad4b-e970-4468-b440-05e149c4538a.jpeg)

### Additional information

https://hastebin.com/share/gidalisuli.makefile

### Example

https://hastebin.com/share/gidalisuli.makefile

**Comment by amyegan** [comment, marked as answer]
Thanks for letting us know about this, Guillermo! We'll work on getting the template updated to fix this problem. 

To help you get un-blocked as quickly as possible, there are a couple of options. I was able to get a test deployment to work by upgrading `@portabletext/react` to version `2`. I also had success using the very similar [Blog with Native Content Editing](https://vercel.com/templates/next.js/blog-nextjs-sanity) template
